python/huggingfaceserver/huggingfaceserver/model.py

```diff
@@ -73,7 +70,11 @@ def __init__(self, model_name: str, kwargs,
         self.tokenizer = None
         self.model = None
         self.mapping = None
-        self.engine = AsyncLLMEngine.from_engine_args(engine_args) if _vllm else None
+        self.vllm_engine = None
+        self.use_vllm = kwargs.get('use_vllm', False)
```

Comment:
should we make it as a default? 
cc @yuzisun

Reply:
updated to set True by default